crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -41,6 +41,19 @@ def roles_for_select(selected)
     )
   end
 
+  # Possible values for nodes default file system
+  def fs_for_select(selected)
+    options_for_select(
+      [
+        ["ext3", "ext3"],
```

Comment:
I think they're all supported by SLES12, so my only concern is really performance impacts that btrfs could have if we don't tweak anything for it in our product. Not really sure how to phrase it, though :/